README.md

```diff
@@ -1 +1,40 @@
-"# math-magicians" 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magicians
+
+> A 3 page project that presents a math calculator with various
+
+## Built With
+
+- React
+
+## Live Demo
+
+Not yet
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
- [ ] Kindly remove this part I highlighted and instead give an instruction on how to clone your repository and set up your project locally.